Lib/multiprocessing/pool.py

```diff
@@ -497,6 +499,7 @@ def _handle_results(outqueue, get, cache):
 
             if task is None:
                 util.debug('result handler ignoring extra sentinel')
+                sentinel_counter += 1
```

Comment:
This extra sentinel is no longer ignored, so the debug is wrong.

Reply:
Obviously right. Fixed that.